### PR TITLE
fix: match subdir under node_module  in transpile

### PR DESCRIPTION
**Pull request by clarkdo**
## Types of changes

- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/webpack/src/config/base.js [hunk 1]
```diff
@@ -225,8 +225,10 @@ export default class WebpackBaseConfig {
       {
         test: /\.jsx?$/i,
         exclude: (file) => {
+          file = file.split('node_modules', 2)[1]
```
Comment (manniL) [inline packages/webpack/src/config/base.js]: What if `node_modules` is not included in that path? 🤔

(Is that even possible?)

Comment (clarkdo) [reply packages/webpack/src/config/base.js]: If it doesn't include `node_modules`, then it won't be included in loader, same as before

**Review by atinux — APPROVED** [review]

**Comment by Haroenv** [comment]
Thanks @clarkdo, @manniL and of course also @Atinux for finding this! I was staring at the wrong things altogether :)

**Comment by clarkdo** [comment]
@Haroenv You're welcome, thank you so much for your interest in Nuxt.js

**Pull request merged**